.github/workflows/bump-version.yaml

```diff
@@ -1,21 +1,14 @@
 name: Bump version and changelog
 on:
-  workflow_dispatch:
-    inputs:
-      version:
-        description: 'The version to release (e.g 1.2.3)'
-        required: true
-        default: ''
-        type: string
-
+  workflow_dispatch
```

Comment:
This seems to be the way to do this without inputs: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch